index.html

```diff
@@ -63,7 +63,7 @@ <h1>Game of Life</h1>
 
         <div>
           <label for="grid-size">Grid Size: </label>
-          <input type="range" id="grid-size" min="25" max="50" />
+          <input type="range" id="grid-size" min="25" max="50" value="30">
```

Comment:
I added value attribute to make it more explicit. It only sets it the first time and then otherwise runs the same. 
Without it it just splits the difference between min/max, so 38.
However, I feel that having it be more explicit is better (bc it's not stated anywhere in either the html or JS, you just jave to know that about <range>)

Reply:
Also, could add boardSize to localStorage to restore settings (just save it for angular)